src/components/totals/Totals.tsx

```diff
@@ -0,0 +1,199 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @copyright Copyright (c) 2021. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+import { Button } from '@invoiceninja/forms';
+import { AxiosResponse } from 'axios';
+import { endpoint, request } from 'common/helpers';
+import { defaultHeaders } from 'common/queries/common/headers';
+import Chart from 'components/charts/Chart';
+
+import React, { useEffect, useState } from 'react';
+
+import Total from './Total';
+import {
+  ChartDataDTO,
+  RequestBodyDTO,
+  TotalDataDTO,
+} from '../../common/dtos/TotalsDTO';
+import { Spinner } from 'components/Spinner';
+import DropdownDateRangePicker from '../DropdownDateRangePicker';
+import { Card } from '@invoiceninja/cards';
+
+export default function Totals() {
```

Comment:
This is off. Through the project, I think I never exported components as defaults. Please use named exports.